src/style.css

```diff
@@ -0,0 +1,32 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 96px;
```

Comment:
check out about [`min-content`](https://developer.mozilla.org/en-US/docs/Web/CSS/min-content)

Reply:
Overall, I can add margins to create gaps between stars, but this looks like "bicycle", as soon as we can't use "gap" attribute in flex container (task restriction).

Reply:
old engine on tests 🥲🥲🥲